### PR TITLE
update moderator staging secrets with secrets for email -smtp and api

### PR DESCRIPTION
**Pull request by cmharlow**
JIRA ticket: https://jira.mozilla.com/browse/SE-1704

Related PRs: https://github.com/mozilla-it/moderator-infra/pull/8

What this PR does: adds in email secrets for Staging. Production SES is still under construction (see PR above). The secrets added via SealedSecret were tested & working as environment variables in the following scripts: https://gist.github.com/cmharlow/e7778ce840633d2112c4dd2bea3acaf1 Included the option for send via API/SDK or SMTP, whichever yall prefer. They'll get loaded as envvars into the deployed application's pod/container.

### CODE REVIEW

File: kubernetes/stage/secret.yaml [hunk 1]
```diff
@@ -6,15 +6,20 @@ metadata:
...
+    PASSWORD_SMTP: AgCO1sgB5IEW31thTNnh/6I/sd7XX1bnKyS0PeZ1kPXbhzMwTxk0cFC5ng5d/K4I3Msxxp/C5+MUbUNo+7sjGVrmdGNvsc73VQlgNucNQBjxHcBgGaFpqGLefaknz9NtUzM3u9/dKYESv6GwAxWkqeUEErHkJFVnQ4SaCV97NeNMaA7JyZzGu5Ehd18oSYKAUQ4R9bL7YU7RXGtKuWDkofIv5CFYEHXiD74JUfSMDpxsfWOwItu0mmm6AYLDbHw/BKt3oF/9fZ/Q2r4s+GjkKEDAol/UbZrZ19ctKXBzL2KRRfgNMqBK5CEKKRstmaMnrEFBBOfyR7hu6K17+pMHeFWbArU1/+4SrdXt65Yri9PepZF/1YzSGxs53eOwg4f2RRgI0ra/cLvb9df8fk0saL8I5akivyjrcqIAFp8cE7AjUP1aD2kl0EAQ4QvQiodAIFPvHPQ88uJt7k5IKVUu/gxKGksQg0eiRKGLcz5F+2FuakH/varrQgBfQcV2YbIBgrIrqbKa3njPS8lUQ/gEUaTCyB+Xr9bC9rl25UWbdlfNEIZ3+TFXd50eBPMcxxY8Oha5nVOZcph9CY8t0Zq3E411UJHqrb32VPloonh6AiJzm5wsZJ/tEun5uQ58nSlSwBI+8FWDEuUkVIg/ZjfeylZSlBrhEnYrTqd9qnISMBE+axJW3TWNl/C6W6zv4SbG178zSuFcHcAZJXj0LRYK/E580t8m/Ge6STVVlDLIbnX39jp8ZqitewRG2u7R3Q==
```
Comment (akatsoulas) [inline kubernetes/stage/secret.yaml:16]: super nit: I would have gone for `SMTP_USERNAME`, `SMTP_HOST` and `SMTP_PASSWORD` but if this is the order the var it's expected it's totally fine

**Review by akatsoulas — APPROVED** [review]
looks good to me :rocket:

File: kubernetes/stage/secret.yaml (hunk 1 above)
Comment (cmharlow) [reply kubernetes/stage/secret.yaml:16]: yeah, tbh, i thought about that too - i think i inverted bc of the samples though & no real other reason. Let me go ahead and set it straight now so it lists alphabetically in a more useful way. Thanks!

Comment (cmharlow) [reply kubernetes/stage/secret.yaml]: changed, and thanks again

**Pull request merged**